src/osd/osd_types.h

```diff
@@ -164,7 +164,7 @@ struct osd_reqid_t {
   {}
 
   DENC(osd_reqid_t, v, p) {
-    DENC_START(2, 2, p);
+    DENC_START_COMPAT_2(2, 2, p);
```

Comment:
I'm afraid `osd_reqid_t` is exposed also to clients (see `osdc`), so we cannot count on the `n+2` rule in this particular case.